week_3/2018202181SZH/readme.txt

```diff
@@ -0,0 +1,16 @@
+���ĵ����ڽ��ʹ���˼·������˼·���Լ����н��

```

Comment:
Save your text files with Chinese characters with UTF-8 encoding please, otherwise GitHub will display it as '??????'.